stockflux-securities-master/src/components/InputForm.js

```diff
@@ -13,9 +17,15 @@ const InputForm = ({ match }) => {
   const [symbol, setSymbol] = useState("");
   const [visible, setVisible] = useState(false);
   const [enabled, setEnabled] = useState(false);
-  const [inProgress, setInProgress] = useState(false);
-  const [isLoading, setIsLoading] = useState(false);
-  const [errorMessage, setErrorMessage] = useState(null);
+  const [contextOrState, setContextOrState] = useState(null);
```

Comment:
When I said context or state, I meant one or the other. You should change this state item and its associated usage to be one or the other. Maybe just go with `formState` to avoid any confusion.

Reply:
So do you still want the same state object to be used for loading, sending, error and success?

Reply:
I just mean rename `contextOrState` to `context` or `state`. My suggestion of using `formState` was just to try and clarify what I mean with a more descriptive name. You don't need to change any logic, just change the name of the variable and its associated setter function.

Reply:
Okay thanks for the clarification, I've now made the change.